TestON/drivers/common/cli/emulator/remotemininetdriver.py

```diff
@@ -376,9 +376,20 @@ def disconnect( self ):
         if self.handle:
             # Close the ssh connection
             self.handle.sendline( "" )
-            self.handle.expect( "\$" )
-            self.handle.sendline( "exit" )
-            self.handle.expect( "closed" )
+            #self.handle.expect( "\$" )
+            i = self.handle.expect( [ '\$', 'mininet>', pexpect.TIMEOUT ],
```

Comment:
Yes, I am using lincoedriver for its purpose. However, packet optical requires a python mininet script (optical.py) from the shell prompt which is provided by the remotemininetdriver (lincoedriver's connect function gives lincoe prompt). 
As mentioned, I am not spawning a mininet topology from remotemininetdriver instead I am using it to run a python script.
Additionally, I don't see a harm in checking the disconnect() if it at the shell prompt or at the mininet prompt.